src/lib.rs

```diff
@@ -477,12 +477,18 @@ impl OptUdeps {
 
 		let mut outcome = Outcome::default();
 
+		let included_packages = compile_opts.spec.get_packages(&ws)?.iter().map(|x|x.package_id()).collect::<HashSet<_>>();
```

Comment:
Can you put the iter, map and collect calls onto the next few lines?

Reply:
sure! like this?